packages/react-storage/src/components/StorageBrowser/context/actions/listLocationsAction.ts

```diff
@@ -40,23 +45,41 @@ export const createListLocationsAction = (
 ): ListLocationsAction =>
   async function listLocationsAction(prevState, input) {
     const { options } = input ?? {};
-    const { exclude, nextToken, pageSize, refresh, reset } = options ?? {};
+    const {
+      exclude,
+      nextToken,
+      pageSize = PAGE_SIZE,
+      refresh,
+      reset,
+    } = options ?? {};
 
     if (reset) {
       return { result: [], nextToken: undefined };
     }
 
-    const output = await listLocations(
-      refresh ? { pageSize } : { nextToken, pageSize }
-    );
+    let locationsResult: ListLocationsOutput['locations'] = [];
+    let nextPageToken: ListLocationsOutput['nextToken'] = refresh
```

Comment:
`nextPageToken` reads as though we are tracking a `pageToken`, prefer `nextNextToken` 😄

Reply:
oh missed pushing this, updated